tests/unit_tests/data_steward/gcloud/bq/bq_test.py

```diff
@@ -40,6 +45,109 @@ def _get_all_field_types(self,) -> typing.FrozenSet[str]:
                             all_field_types.add(field.get('type'))
         return frozenset(all_field_types)
 
+    def list_item_from_table_id(self, table_id: str) -> TableListItem:
+        """
+        Get a table list item as returned by :meth:`bigquery.Client.list_tables` 
+        from a table ID
+        
+        :param table_id: A table ID including project ID, dataset ID, and table ID, 
```

Comment:
nit: For these I'd suggest using `fq_table_id` instead of just `table_id`. This refers to the fully qualified table ID as `project_id.dataset_id.table_id`.
```suggestion
    def list_item_from_table_id(self, fq_table_id: str) -> TableListItem:
        """
        Get a table list item as returned by :meth:`bigquery.Client.list_tables` 
        from a fully qualified table ID
        
        :param fq_table_id: A fully qualified table ID including project ID, dataset ID, and table ID, 
```